src/unix/notbsd/linux/musl/b64/mod.rs

```diff
@@ -114,6 +114,13 @@ s! {
         pub msg_flags: ::c_int,
     }
 
+    pub struct cmsghdr {
+        pub cmsg_len: ::socklen_t,
+        pub __pad1: ::c_int,
```

Comment:
I think it should. After all it's a C binding, so people can do whatever they want.